config/locales/server.en.yml

```diff
@@ -2818,7 +2819,7 @@ en:
       clean_up_inactive_users_after_days: "deactivated|inactive|unactivated"
       navigation_menu: "sidebar|header dropdown"
       purge_unactivated_users_grace_period_days: "deactivated|inactive|unactivated"
-      rich_editor: "rte|rich text|composer"
+      default_composition_mode: "rte|rich text|composer|markdown"
```

Comment:
is it composable, e.g. `rte|rich|text|editor|composer|markdown` will match both `rich editor` and `rich text`?

Reply:
Not sure...will add extra keywords jist in case